src/Lean/Compiler/IR/EmitC.lean

```diff
@@ -110,15 +113,15 @@ def emitFnDeclAux (decl : Decl) (cppBaseName : String) (addExternForConsts : Boo
     emit ")"
   emitLn ";"
 
-def emitFnDecl (decl : Decl) (addExternForConsts : Bool) : M Unit := do
+def emitFnDecl (decl : Decl) (isExternal : Bool) : M Unit := do
   let cppBaseName ← toCName decl.name
-  emitFnDeclAux decl cppBaseName addExternForConsts
+  emitFnDeclAux decl cppBaseName isExternal
 
 def emitExternDeclAux (decl : Decl) (cNameStr : String) : M Unit := do
   let cName := Name.mkSimple cNameStr
   let env ← getEnv
   let extC := isExternC env decl.name
-  emitFnDeclAux decl cNameStr (!extC)
+  emitFnDeclAux decl cNameStr extC
```

Comment:
@leodemoura It makes more sense this way around, right? We want to annotate `[extern "..."]` constants with `extern`, but not `[extern inline ...]` ones. It doesn't look like this was used before, but now that I extended the parameter to functions, it is.